third_party/CMakeLists.txt

```diff
@@ -297,3 +297,13 @@ target_compile_definitions(
   tpcds_dbgen
   PUBLIC
   -DLINUX)
+
+target_compile_options(
+  tpcds_dbgen
+  PRIVATE
+
+  -Wno-implicit-int
```

Comment:
Any meaning behind the order? Shall we sort by name?

Reply:
Will do, but I'd like for the current CI build to complete